Linkta-org/core
# Feature/auth client views

This pull request updates the placeholder components for SignInPage, SignUpPage, ForgotPasswordPage, and UpdatePasswordPage. These are all unprotected routes, meaning the views can be accessed by a user that is not authenticated.

Note: The four views referenced above are similar to each other, and are sharing a single stylesheet.

In addition, this PR provides an alternate static layout to the "MainLayout" which is a protected route. The "UnauthorizedLayout" component is a simplified layout with no side navigation.

Note: There is no logic included in this PR. The buttons and forms are not currently functioning.

This PR also includes updates to the "HomePage" component to make it closely resemble the Figma design.

Finally, this PR includes a new Linkta logo image. This file is not meant to be permanent - there has been a long-lived issue with the existing logo file where the transparent background is not completely transparent, and is clearly visible against a dark green background. I'm open to suggestions regarding a permanent logo but we should not ignore images with problems such as visible artifacts or low resolution.



Fixes #241 #243 #244

## Type of change





## How Can this be tested? Testing Plan to review this PR



- [x] Where in the app does this change take place? | Most changes are found in client/features/auth-pages
- [x] What are the steps to test this change? | Visit client/hooks/useAuth.tsx AND client/routes/index.tsx and change the value of "isAuthenticated" to "false" - this prevents navigation to the protected routes and redirects to /userLogin. Additional auth routes are /userSignup, forgotPassword, and updatePassword.
- [x] What is the expected output? | Quality views that resemble the Figma design

The biggest risk that I can see is that some views could be too tall for some screens. I have styled to ensure that the tallest auth view (userSignup) is compatible with a 720p display in full-screen mode.

![chrome_92pmmyG75W](https://github.com/Linkta-org/core/assets/71753585/d7539c5d-7396-40dd-b39d-85d076b028ae)


## Checklist:
